Source/i_main.c

```diff
@@ -87,7 +87,7 @@ void I_AtExit(atexit_func_t func, boolean run_on_error)
 {
     atexit_listentry_t *entry;
 
-    entry = malloc(sizeof(*entry));
+    entry = (malloc)(sizeof(*entry));
```

Comment:
Not relevant to this PR, but how do you feel about removing memory function macros? I think they don't help and are actually very annoying.

Reply:
Yes, they often feel like shooting into the own foot. We would need to find out how often they are used implicitely in the code, though, i.e. by calling `malloc()` when indeed `Z_Malloc()` was meant.

Reply:
These macros are the bane of my existence. There are places calling just malloc which is actually Z_Malloc but another file explicitly calls Z_Free on that data. Or vice versa, where something just calls free on another thing that was allocated with Z_Malloc. I had a branch adjusting all of this in dsda-doom but in the end I didn't want to risk having a mismatch in some context I wasn't aware of. It's probably cleaner in woof though.